### DISCUSSION TITLE
Storage limit exceeded

### DISCUSSION THREAD

**Discussion by bobaikato** [community/community#45126] [Actions]
Hey, i got this email stating that I have used 100% of included services for GitHub Storage (GitHub Actions and
Packages).
To continue using Actions & Packages uninterrupted, update your spending limit.
Your usage will reset on February 23, 2023.

Not sure where to clean up this storage  and also what exactly is taking this space?

Please help!

**Comment by jge162** [comment]
@bobaikato To figure out what is taking up the storage space, you can check the "Packages" section of your GitHub account. This will show you the size of each package and allow you to delete any that are no longer needed. Additionally, you can check the "Actions" section of your account to see if there are any large artifacts or logs that can be deleted.

**Reply by bobaikato** [reply]
That's not particularly helpful.

Here is the package section for our companies page
![image](https://user-images.githubusercontent.com/14136806/214009187-f54cdf43-2e94-4aae-acc5-bebcf409cf4e.png)

Same with actions
![image](https://user-images.githubusercontent.com/14136806/214009627-e33601b3-5a3b-435b-b674-2c7c7040737a.png)

Can you be a little clear with a guide on your recommendations, please?

**Reply by jge162** [reply]
@bobaikato

1. Check your repository settings for each repository in your account. Go to the "Insights" tab and then the "Storage" option. This will show you the size of each branch and repository, and allow you to delete any large files that are no longer needed.

2. Check for large files in your repository by using the command "git lfs ls-files" in your local repository. This will show you all the large files in your repository and their corresponding sizes.

3. Review the "Actions" tab in your GitHub account and look for any large artifacts or logs that can be deleted.

**Reply by jge162** [reply]
to use git do this @bobaikato 


Make sure you have Git LFS installed on your local machine. 
You can install it by following the instructions on the Git LFS website: https://git-lfs.github.com/

Clone the repository you want to check for large files on your local machine using the command: 
```python
git clone https://github.com/[username]/[repository].git
```
Navigate to the root directory of the repository on your local machine using the command: 
```python
cd [repository]
```
Run the command 
```python
git lfs ls-files
```

 in the root directory of the repository to see a list of all the large files 
tracked by Git LFS and their corresponding sizes.

**Reply by jge162** [reply]
this will work 100%

**Reply by bobaikato** [reply]
We don't have a single large files pushed. None, that's confirmed already

From the email, the storage usage is related to GitHub Actions and Packages. I don't see where and how we could have used that amount of storage, and if so, we need a way to clean.

We don't store any package, either. All we use GitHub action is to build and deploy to Azure.

**Reply by jge162** [reply]
check the logs of your GitHub Actions to see if there are any large files being pushed or stored in your repository. 

Check the Azure storage to confirm if the storage is being used there.

if you did not do LFS then you can't say you are not using storage. you got the email from them. you should check.

**Reply by jge162** [reply]
Hopefully you find it. sorry this is happening .
